xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -59,10 +59,23 @@ void CGUIWindowPVRGuide::OnInitWindow()
 
   CGUIEPGGridContainer *epgGridContainer =
     dynamic_cast<CGUIEPGGridContainer*>(GetControl(m_viewControl.GetCurrentControl()));
+
   if (epgGridContainer)
-    epgGridContainer->GoToNow();
+  {
+    CPVRChannelGroupPtr group(GetGroup());
+    if (!group)
+      group = g_PVRManager.GetPlayingGroup(m_bRadio);
```

Comment:
Okay, not really nice but for now we can live with this. But you need to adjust the code, because `GetGroup()` could return an outdated group e.g. if you change the group within the channel osd. You should always use the playing group on init of the window as we do in the base class.

```
CPVRChannelGroupPtr group(g_PVRManager.GetPlayingGroup(m_bRadio));
```